.github/workflows/nerdctl.yaml

```diff
@@ -52,17 +50,19 @@ jobs:
 
       - name: Install nerdctl
         run: |
+          # Remove Docker and Podman
           sudo systemctl is-active --quiet docker.service || systemctl stop docker.service
-          sudo apt-get remove -y docker-ce docker-ce-cli docker-buildx-plugin podman
-          # Install Containerd
-          sudo curl -sSL https://github.com/containerd/containerd/releases/download/v${CONTAINERD_VERSION}/containerd-${CONTAINERD_VERSION}-linux-amd64.tar.gz|sudo tar -xvz -C /usr
-          sudo systemctl restart containerd.service
+          sudo apt-get remove -y docker-ce docker-ce-cli podman containerd.io
+          # Install nerdctl full package
+          sudo curl -sSL https://github.com/containerd/nerdctl/releases/download/v${NERDCTL_VERSION}/nerdctl-full-${NERDCTL_VERSION}-linux-amd64.tar.gz | sudo tar -xvz -C /usr/local
+          # Start Containerd
+          sudo curl -sSL https://raw.githubusercontent.com/containerd/containerd/main/containerd.service > containerd.service
+          sudo mv containerd.service /etc/systemd/system
+          sudo systemctl daemon-reload
+          sudo systemctl enable --now containerd
+          # Show Versions
           sudo ctr version
-          # Install CNI
```

Comment:
why are these no longer required?

Reply:
HI @aojea 

Because the nerdctl-full contains the CNI dependency, it also includes:

```
# nerdctl (full distribution)
- nerdctl: v1.7.4
- containerd: v1.7.13
- runc: v1.1.12
- CNI plugins: v1.4.0
- BuildKit: v0.12.5
- Stargz Snapshotter: v0.15.1
- imgcrypt: v1.1.9
- RootlessKit: v2.0.1
- slirp4netns: v1.2.2
- bypass4netns: v0.4.0
- fuse-overlayfs: v1.13
- containerd-fuse-overlayfs: v1.0.8
- Kubo (IPFS): v0.26.0
- Tini: v0.19.0
- buildg: v0.4.1
- ```

ref to: https://github.com/containerd/nerdctl/releases/tag/v1.7.4

Using the `nerdctl-full` can make the CI easy to maintaince, because the nerdctl manage the dependency version mapping.